internal/command/logs/shipper_setup.go

```diff
@@ -147,40 +123,55 @@ func runSetup(ctx context.Context) (err error) {
 			CPUs:     1,
 			MemoryMB: 256,
 		},
-		Image: "flyio/log-shipper",
+		Image: "flyio/log-shipper:auto-d5a96e6",
 	}
 
 	machines, err := flapsClient.List(ctx, "")
 
+	if err != nil {
+		return err
+	}
+
 	launchInput := api.LaunchMachineInput{
 		AppID:  shipperApp.Name,
 		Name:   "log-shipper",
 		Config: machineConf,
 	}
 
-	// We already have a log shipper VM, so just update it in-place to pick up any new secrets
+	var machine *api.Machine
+
 	if len(machines) > 0 {
-		fmt.Fprintf(io.Out, "Restarting machine %s\n", machines[0].ID)
-		machine := machine.NewLeasableMachine(flapsClient, io, machines[0])
-		machine.AcquireLease(ctx, time.Second*5)
-		launchInput.ID = machines[0].ID
-		launchInput.Config = machines[0].Config
-		machine.Update(ctx, launchInput)
-		machine.ReleaseLease(ctx)
-		return
-	}
+		machine = machines[0]
+	} else {
 
-	regionResponse, err := gql.GetNearestRegion(ctx, client)
+		regionResponse, err := gql.GetNearestRegion(ctx, client)
 
-	if err != nil {
-		return err
-	}
+		if err != nil {
+			return err
+		}
 
-	launchInput.Region = regionResponse.NearestRegion.Code
+		launchInput.Region = regionResponse.NearestRegion.Code
 
-	machine, err := flapsClient.Launch(ctx, launchInput)
+		machine, err := flapsClient.Launch(ctx, launchInput)
 
-	fmt.Fprintf(io.Out, "Launched machine %s\n in the %s region", machine.ID, launchInput.Region)
+		if err != nil {
+			return err
+		}
 
+		fmt.Fprintf(io.Out, "Launched log shipper VM %s\n in the %s region", machine.ID, launchInput.Region)
+	}
+
+	cmd := []string{"/add-logger.sh", targetApp.Name, "logtail", "'" + tokenResponse.CreateLimitedAccessToken.LimitedAccessToken.TokenHeader + "'", logtailToken}
```

Comment:
It would be a bit more robust to use something like [shellescape](https://pkg.go.dev/github.com/alessio/shellescape) for this.